src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Converters/TimeIdentifierCategoryJsonConverterTests.cs

```diff
@@ -0,0 +1,123 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Converters;
+using GovUk.Education.ExploreEducationStatistics.Common.Model;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Converters
+{
+    public class TimeIdentifierCategoryJsonConverterTests
```

Comment:
Would be good to see an example in here of a TimeIdentifierCategory being converted to and from JSON, if that is what this test is testing

Reply:
you mean going from object => json => object? Sounds like a good idea. I'll get on it.

Reply:
Done